### DISCUSSION TITLE
Focal crop, does not work according to documentation

### DISCUSSION THREAD

**Discussion by sarabando** [spatie/image#245] [General]
According documentation for focal cropping [resizing-images](https://spatie.be/docs/image/v3/image-manipulations/resizing-images)
I must pass the percentages from 0 - 100 but it doesn't work. 

In the previous version 2.x this functionality worked without problems,
According to the tests I carried out, it is necessary to convert the percentage into absolute positions.

**Comment by dualhub** [comment]
I'm also struggling with this. A bit disappointed that there's no response from Spatie. The documentation is kind of unclear. 
Any help, nice Spatie people?

**Comment by GarethSomers** [comment]
Hey @sarabando and @dualhub I've created a [PR](https://github.com/spatie/image/pull/301) to target this problem.

It's intended to match the `focalCrop` functionality in v2, where we can pass percentages for the focal point, and otherwise matching the [Fit:Max behaviour](https://spatie.be/docs/image/v3/image-manipulations/resizing-images#content-fitmax).
